src/OSPSuite.Infrastructure.Import/Core/DataFormat/AbstractColumnsDataFormat.cs

```diff
@@ -60,7 +60,7 @@ private void setDimensionsForMappings(IReadOnlyList<ColumnInfo> columnInfos)
             if (mappedColumn?.Unit == null || mappedColumn?.Dimension != null)
                continue;
 
-            var concreteColumnInfo = columnInfos.First(x => x.Name == mappedColumn.Name);
+            var concreteColumnInfo = columnInfos.First(x => x.DisplayName == mappedColumn.Name);
```

Comment:
What's the displayName of a column Info? Do we have instances where it's not the same as Name? I think this makes sense to use DisplayName but I don't know why we have this in the first place